library/src/main/java/com/google/android/exoplayer2/extractor/flv/ScriptTagPayloadReader.java

```diff
@@ -215,4 +244,19 @@ private static Object readAmfData(ParsableByteArray data, int type) {
     }
   }
 
+  public void setKeyFrameTimes(List<Double> keyFrameTimes) {
```

Comment:
I'm unclear why you need these setter methods. Why can't you just assign the variables directly?